pkgs/by-name/pr/prometheus-kafka-adapter/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+}:
+
+buildGoModule rec {
+  pname = "prometheus-kafka-adapter";
+  version = "1.9.0";
+
+  src = fetchFromGitHub {
+    owner = "Telefonica";
+    repo = "prometheus-kafka-adapter";
+    rev = version;
+    hash = "sha256-nL6NSLG2POgnIvaQ7JWahKCnnOh3/ZatM22kxZ6CNTQ=";
+  };
+
+  vendorHash = null;
+
+  passthru.tests.default = nixosTests.prometheus-kafka-adapter;
+
+  meta = with lib; {
+    description = "Prometheus remote write adapter for Apache Kafka";
+    homepage = "https://github.com/Telefonica/prometheus-kafka-adapter";
+    changelog = "https://github.com/Telefonica/prometheus-kafka-adapter/releases/tag/${src.rev}";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ illustris ];
+    mainProgram = pname;
```

```suggestion
    mainProgram = "prometheus-kafka-adapter";
```
this shouldn't be done

---

This is for brevity. Is there any good reason for this change?

---

I'll take that as a no